docs/howto/connection/oracle.rst

```diff
@@ -85,3 +85,36 @@ Extra (optional)
     .. code-block:: bash
 
         export AIRFLOW_CONN_ORACLE_DEFAULT='oracle://oracle_user:XXXXXXXXXXXX@1.1.1.1:1521?encoding=UTF-8&nencoding=UTF-8&threaded=False&events=False&mode=sysdba&purity=new'
+
+
+Tips and Tricks for Oracle Shops
+================================
+
+Oracle users an SID to identify a unique database instance. Oracle also has an unusual definition of schema and user, wherein a schema set of objects owned by a user. Therefore, creating a connection through the Airflow web GUI can be confusing.
+
+To create a connection: 
+
+- Go to Admin => Connections => Create
+
+- Name your connection
+
+- Select connection type: Oracle
+
+- Enter your hostname in the following format: NameOrIP:PORT#/SID Example(s): 10.0.0.111:1521/orcl or weshouldhaveusedpostgres.ourdomain.com:1521/dev
```

Comment:
I wonder if it makes sense to add these sections. This greatly duplicates the description in the section above or [the page one level higher](https://airflow.readthedocs.io/en/latest/howto/connection/index.html). What do you think? Is this information duplicated or maybe I missed something?

Reply:
I see what you mean, I guess people that come straight to this Oracle doc can see this. It doesn't hurt to add it as there is some extra info on the hostname format and such.

Reply:
Do we need to add these examples to the new section? I think the section above will be more helpful.

Reply:
The problem is that this section deviates significantly from the standards that are in the rest of the other sections for no good reason  This makes it more difficult to maintain and harder to use and read

Reply:
Do you think I should just make a reference for [the page above](https://airflow.readthedocs.io/en/latest/howto/connection/index.html) instead of adding the examples then?

Reply:
Yeah. Let's simplify and point there. It turned out that "move" = "delete" and point to the right place.